1.1/functions.ttl

```diff
@@ -0,0 +1,27 @@
+PREFIX safuncs: <http://www.opengis.net/def/function/aggregate/geosparql/>
+PREFIX funcs: <http://www.opengis.net/def/function/geosparql/>
+
+funcs: a skos:Collection ;
```

Comment:
You need a ConceptScheme & Concept declarations as well as the Collections with `skos:member` elements so that this vocabulary can be properly picked up by the vocab publication tools (not just pyLODE but also the OGC's vocab server).